features/manage-position/Deposit.tsx

```diff
@@ -7,10 +7,17 @@ import EmpContract from "../../containers/EmpContract";
 import Collateral from "../../containers/Collateral";
 import Position from "../../containers/Position";
 
+import { useEtherscanUrl } from "./useEtherscanUrl";
+
 const Container = styled(Box)`
   max-width: 720px;
 `;
 
+const Link = styled.a`
```

Comment:
@adrianmcli we should move these css/jsx blocks to a shared library

Reply:
yeah, but to be honest I'm hesitant to spend much more time on this unless we get the go-ahead to focus on this as a product.

Reply:
great point